java.lang.NullPointerException: Cannot invoke "java.util.concurrent.CompletableFuture.thenRun(java.lang.Runnable)" because "this.initializeFuture" is null

Using 1.16.0.329 version. Getting the following error while working with the Quarkus project in https://github.com/quarkusio/registry.quarkus.io

```
Internal error: java.lang.NullPointerException: Cannot invoke "java.util.concurrent.CompletableFuture.thenRun(java.lang.Runnable)" because "this.initializeFuture" is null

java.util.concurrent.CompletionException: java.lang.NullPointerException: Cannot invoke "java.util.concurrent.CompletableFuture.thenRun(java.lang.Runnable)" because "this.initializeFuture" is null
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.concurrent.CompletableFuture.thenRun(java.lang.Runnable)" because "this.initializeFuture" is null
	at com.redhat.devtools.intellij.quarkus.lsp4ij.LanguageServerWrapper.registerCapability(LanguageServerWrapper.java:727)
	at com.redhat.devtools.intellij.quarkus.lsp4ij.LanguageClientImpl.lambda$registerCapability$2(LanguageClientImpl.java:85)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	... 6 more
```

--- a/src/main/java/com/redhat/devtools/intellij/quarkus/lsp4ij/LanguageServerWrapper.java
+++ b/src/main/java/com/redhat/devtools/intellij/quarkus/lsp4ij/LanguageServerWrapper.java
@@ -323,7 +323,11 @@ public class LanguageServerWrapper {
             initializeFuture = languageServer.initialize(initParams).thenAccept(res -> {
                 serverCapabilities = res.getCapabilities();
                 this.initiallySupportsWorkspaceFolders = supportsWorkspaceFolders(serverCapabilities);
-            }).thenRun(() -> {
+            });
+            initializeFuture.thenRun(() -> {
+            	// Here we call languageServer.initialized which will send to this IJ LSP client several 'client/registerCapability' 
+            	// which will call the private method registerCapability(RegistrationParams params)
+            	// See https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#client_registerCapability
                 this.languageServer.initialized(new InitializedParams());
             });
 

